pr_insight/algo/git_patch_processing.py

```
@@ -325,18 +326,19 @@ def convert_to_hunks_with_lines_numbers(patch: str, file) -> str:
             is_plus_lines = any([line.startswith('+') for line in new_content_lines])
-            if is_plus_lines:
-                patch_with_lines_str = patch_with_lines_str.rstrip() + '\n__new hunk__\n'
-                for i, line_new in enumerate(new_content_lines):
-                    patch_with_lines_str += f"{start2 + i} {line_new}\n"
         if old_content_lines:
             is_minus_lines = any([line.startswith('-') for line in old_content_lines])
-            if is_minus_lines:
-                patch_with_lines_str = patch_with_lines_str.rstrip() + '\n__old hunk__\n'
-                for line_old in old_content_lines:
-                    patch_with_lines_str += f"{line_old}\n"
+        if is_plus_lines or is_minus_lines:  # notice 'True' here - we always present __new hunk__ for section, otherwise LLM gets confused
```

Ruff

Line too long (139 > 120)

(E501)